x-pack/plugins/observability/public/components/alerts_table/get_alerts_table_configuration.tsx

```diff
@@ -53,7 +53,6 @@ export const getAlertsTableConfiguration = (
             []
           )}
           ecsData={{ _id: alert._id, _index: alert._index }}
-          eventId={alert._id}
```

Comment:
Why did we have `eventId` previously?

Reply:
I don't know, but it's not used anymore. Probably left over from a previous implementation.